src/account.js

```diff
@@ -0,0 +1,23 @@
+
+/* Account login state manager
+ */
+
+import _ from 'lodash'
+import db from './db'
+import { storeKey } from './config'
+
+export default { setUser }
+
+function setUser(user) {
+  storeKey('user', user.id)
+  storeKey('remote', user.couch.url)
+  const opts = { include_docs: true }
+  return db.allDocs(opts)
+  .then(rslt => {
+    const author = { user: user.id }
+    const docs = _.map(rslt.rows, 'doc')
```

Comment:
I didn't do this because I thought it looked a more ugly, but didn't consider memory efficiency. Good point. I'll update it to use a chain!